src/Employer/Employer.Web/Views/VacancyPreview/AdvertPreview.cshtml

```diff
@@ -10,22 +10,32 @@
   <a asp-route="@RouteNames.EmployerCheckYourAnswersGet" asp-all-route-data="@Model.RouteDictionary" esfa-automation="link-back" class="govuk-back-link">Back to check your answers</a>
 }
 
-<h1 class="govuk-heading-xl">Preview your advert</h1>
+<div class="govuk-grid-column-two-thirds-from-desktop">
+    <h1 class="govuk-heading-xl">Preview your advert</h1>
+    <div class="govuk-inset-text">
+        Proofread your advert before submitting. We might reject your advert if it has spelling or formatting errors.
```

Comment:
Not sure this matches the copy doc 🤔

Reply:
This is highlighted in blue in v30 of the copy docs, as copy that has been left out. Doesn't that mean it needs to be added?